app/src/docs/v1.api-spec.yaml

```diff
@@ -1231,7 +1233,8 @@ paths:
         need to re-upload preexisting files through the COMS API. This endpoint
         does not guarantee immediately synchronized results. Synchronization
         latency will be affected by the remaining number of objects awaiting
-        synchronization.
+        synchronization. This endpoint updates the last known successful
+        synchronization request date when invoked.
       operationId: syncDefault
```

Comment:
we could add lastSyncRequestedDate to the DB-Bucket schema. same for the object sync date

Reply:
It's already in place from a previous PR.